### PR TITLE
Small fixes for Google provider's system tests

### PR DESCRIPTION
**Pull request by VladaZakharova**
This PR removes prefix 'example_' from DAG name to follow name convention, adds comments and define 'default' ENV_ID if not provided.

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/system/providers/google/cloud/bigquery/example_bigquery_transfer.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
```
Comment (romsharon98) [inline tests/system/providers/google/cloud/bigquery/example_bigquery_transfer.py]: I think you want to change this line.
```suggestion
ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID", "default")
```
you have more places where you miss to add default

**Review by potiuk — COMMENTED** [review]
LGTM besides the comment from @romsharon98

**Review by potiuk — APPROVED** [review]

**Review by romsharon98 — APPROVED** [review]
Looks good to me also, just let me know if you want to fix my comment or not (it depends on you) and will merge it

File: tests/system/providers/google/cloud/bigquery/example_bigquery_transfer.py (hunk 1 above)
Comment (VladaZakharova) [reply tests/system/providers/google/cloud/bigquery/example_bigquery_transfer.py]: Yes, indeed, i fixed some of them, but it is actually good to have but not required for the test execution :D 
Let me add more (there are A LOT of tests without it)

**Pull request merged**